e2e/protractor/suites/pagination/multiple-pages-trash.test.ts

```diff
@@ -1,151 +0,0 @@
-/*!
- * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
- *
- * Alfresco Example Content Application
- *
- * This file is part of the Alfresco Example Content Application.
- * If the software was purchased under a paid Alfresco license, the terms of
- * the paid license agreement will prevail. Otherwise, the software is
- * provided under the following open source license terms:
- *
- * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Lesser General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * The Alfresco Example Content Application is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
- */
-
-import { LoginPage, BrowsingPage, Utils, AdminActions, UserActions, RepoClient } from '@alfresco/aca-testing-shared';
-
-describe('Pagination on multiple pages on Trash', () => {
-  const random = Utils.random();
-
-  const username = `user-${random}`;
-  const filesForDelete = Array(51)
-    .fill('file')
-    .map((name, index): string => `${name}-${index + 1}-${random}.txt`);
-  let filesDeletedIds: string[];
-
-  const userApi = new RepoClient(username, username);
-  const adminApiActions = new AdminActions();
-  const userActions = new UserActions();
-
-  const loginPage = new LoginPage();
-  const page = new BrowsingPage();
-  const { dataTable, pagination } = page;
-
-  beforeAll(async () => {
-    await adminApiActions.createUser({ username });
-
-    filesDeletedIds = (await userApi.nodes.createFiles(filesForDelete)).list.entries.map((entries: any) => entries.entry.id);
-
-    await userActions.login(username, username);
-    await userActions.deleteNodes(filesDeletedIds, false);
-    await userActions.waitForTrashcanSize(51);
```

Comment:
waitForTrashcanSize function can be removed as it is unused now

Reply:
removed